site/docs/api/spring.md

```diff
@@ -328,3 +328,23 @@ attributes.
     </jet:client>
 </beans>
 ```
+
+## SpringBoot Integration
+
+Hazelcast Jet provides
+[Hazelcast Jet Spring Boot Starter](https://github.com/hazelcast/hazelcast-jet-contrib/tree/master/hazelcast-jet-spring-boot-starter)
+which automatically configures and starts a `JetInstance`, either a
+server or a client, if Jet is on the classpath.
+
```

Comment:
can we give an example how to use it?

Reply:
I've added a tutorial and linked it here